docs/installation/integrations/react-multiroot.md

```diff
@@ -0,0 +1,138 @@
+---
+menu-title: Multi-root integration
+meta-title: React rich text editor component | CKEditor 5 documentation
+category: react
+order: 20
+---
+
+{@snippet installation/integrations/framework-integration}
+
+# React rich text multi-root editor hook
+
+<p>
+	<a href="https://www.npmjs.com/package/@ckeditor/ckeditor5-react" target="_blank" rel="noopener">
+		<img src="https://badge.fury.io/js/%40ckeditor%2Fckeditor5-react.svg" alt="npm version" loading="lazy">
+	</a>
+</p>
+
+CKEditor&nbsp;5 consists of {@link installation/getting-started/predefined-builds ready-to-use editor builds} and {@link framework/index CKEditor&nbsp;5 Framework} upon which the builds are based.
+
+The easiest way to use multi-root CKEditor&nbsp;5 in your React application is by using the {@link installation/getting-started/predefined-builds#multi-root-editor multi-root rich text editor build}.
+
+<info-box hint>
+	The multi-root editors in React is supported since version 6.2.0 of this package.
+
+	Unlike the {@link installation/integrations/react default integration}, in the multi-root editor we prepared the integration based on the hooks and new React mechanisms.
+</info-box>
+
+## Quick start
+
+This guide assumes you already have a React project. If you want to create a new one, you can use the [`create-react-app`](https://create-react-app.dev/) CLI. It allows you to create and customize your project with templates. For example, you can set up your project with TypeScript support.
```

Comment:
CRA is deprecated and discouraged right now, let's not treat it as a blocker but we will need to fix it.